frontend/appflowy_flutter/lib/plugins/database_view/widgets/row/cells/select_option_cell/select_option_editor.dart

```diff
@@ -174,17 +206,35 @@ class _TextField extends StatelessWidget {
 }
 
 class _Title extends StatelessWidget {
-  const _Title({Key? key}) : super(key: key);
+  const _Title({Key? key, required this.onPressed}) : super(key: key);
+
+  final void Function() onPressed;
 
   @override
   Widget build(BuildContext context) {
     return Padding(
       padding: const EdgeInsets.symmetric(horizontal: 12.0),
       child: SizedBox(
         height: GridSize.popoverItemHeight,
-        child: FlowyText.medium(
-          LocaleKeys.grid_selectOption_panelTitle.tr(),
-          color: Theme.of(context).hintColor,
+        child: Row(
+          crossAxisAlignment: CrossAxisAlignment.center,
+          children: [
+            FlowyText.medium(
+              LocaleKeys.grid_selectOption_panelTitle.tr(),
+              color: Theme.of(context).hintColor,
+            ),
+            FlowyIconButton(
+              onPressed: onPressed,
+              hoverColor: Colors.transparent,
```

Comment:
Why using `transparent` color here?

Reply:
![image](https://github.com/AppFlowy-IO/AppFlowy/assets/51508126/b64f1ef1-b64e-40b7-a636-becc10f5a270)

because this looks very weird